packages/quantic/force-app/main/default/lwc/quanticFoldedResultList/quanticFoldedResultList.js

```diff
@@ -0,0 +1,158 @@
+import {
+  registerComponentForInit,
+  initializeWithHeadless,
+  getHeadlessBundle,
+} from 'c/quanticHeadlessLoader';
+import {LightningElement, api} from 'lwc';
+
+/** @typedef {import("coveo").FoldedResultList} FoldedResultList */
+/** @typedef {import("coveo").FoldedResultListState} FoldedResultListState */
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+/** @typedef {import("coveo").ResultTemplatesManager} ResultTemplatesManager */
+/** @typedef {import("coveo").ResultsPerPage} ResultsPerPage */
+
+/**
+ * The `QuanticFoldedResultList` component is responsible for displaying query results by applying one or more result templates.
+ * This component can display query results that have a parent-child relationship with any level of nesting.
+ * @fires CustomEvent#registerresulttemplates
+ * @category Search
+ * @example
+ * <c-quantic-folded-result-list engine-id={engineId} fields-to-include="objecttype,gdfiletitle" collection-field="foldingcollection" parent-field="foldingparent" child-field="foldingchild" number-of-folded-results="2"></c-quantic-folded-result-list>
+ */
+export default class QuanticFoldedResultList extends LightningElement {
+  /**
+   * The ID of the engine instance the component registers to.
+   * @api
+   * @type {string}
+   */
+  @api engineId;
+  /**
+   * A list of fields to include in the query results, separated by commas.
+   * @api
+   * @type {string}
+   * @defaultValue `'date,author,source,language,filetype,parents,sfknowledgearticleid'`
+   */
+  @api fieldsToInclude =
+    'date,author,source,language,filetype,parents,sfknowledgearticleid';
+  /**
+   * The name of the field on which to do the folding.
+   * @api
+   * @type {string}
+   * @defaultValue `'foldingcollection'`
+   */
+  @api collectionField;
```

Comment:
Shouldn't `collectionField` be initialized to `foldingcollection`? The same goes for `parentField`, `childField`, and `numberOfFoldedResults`.

Reply:
didn't give these properties any initial values because the default values are used when these properties are `undefined`, but yeah you are right it would be safer to set these initial values directly in Quantic, I will add them.